docs/oauth2-examples-multiresource.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Use multiple OAuth 2.0 servers and/or audiences
+displayed_sidebar: docsSidebar
+---
+<!--
+Copyright (c) 2005-2024 Broadcom. All Rights Reserved. The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Use multiple OAuth 2.0 servers and/or audiences
+
+Demonstrate how to authenticate using OAuth 2.0 protocol
+and several OAuth resources using the following flows:
+
+* Access AMQP protocol
+* Access Management UI
+
+## Prerequisites to follow this guide
+
+- Docker
+
+## Motivation
+
+All the examples demonstrated by this tutorial, except for this use case, configure a single **resource_server_id** and therefore a single **OAuth 2.0 server**.
```

Comment:
Is this a tutorial? If so the H1 should say this?

Reply:
Addressed in a previous commit